examples/hooks/README.md

```diff
@@ -52,16 +57,16 @@ hooks:
   post-deploy:
     - echo "Deployment finished"
     - graphql get-schema --project db
-    - graphql prepare
+    - graphql codegen
 ```
 
 * Printing "Deployment finished"
 
 * Getting the latest schema
 
-* Running `graphql prepare` to generate TS bindings
+* Running `graphql codegen` to generate TS bindings
```

Comment:
add note that prisma-binding 2.0 is needed

Reply:
But this example does not use binding at all and was meant to show how hooks work. Is installing `prisma-binding` mandatory?

This can be an issue in `graphql-cli` as currently it fails silently. Success message does nothing.